### PR TITLE
Bug(?) in Coordinator.withdrawMGN...

### PR DESCRIPTION
**Pull request by Velenir**
Withdrawing through coordinator doesn't withdraw everything:
small decimals of WETH and GNO stay in the pools
MGN isn't withdrawn at all

`npx truffle exec trufflescripts/auction.js --revert` at the end of the pooling produces:
```
...
    Account 0x1dF62f291b2E969fB0849d99D9Ce41e2F137006e
      ETH:  9900.039738296755507331
      GNO:  9900.039643445078847281
      MGN:  103.511891688853728643
      Locked MGN:  0
      Will be unlocked MGN:  0
  
  Pool Balances
    EthPool:
      ETH:  0.000000000000000003
      GNO:  0
      MGN:  0.000000000000000003
      Locked MGN:  0
      Will be unlocked MGN:  0
    GnoPool:
      ETH:  0
      GNO:  0.000000000000000003
      MGN:  0.000000000000000006
      Locked MGN:  0
      Will be unlocked MGN:  0
```

We can see that MGN from the pools was withdrawn into the accounts.
This was done with direct `pool.withdrawMagnolia()` calls

-----------------------------

`npx truffle exec trufflescripts/auction.js --revert -c` at the end of the pooling produces:
```
...
    Account 0x1dF62f291b2E969fB0849d99D9Ce41e2F137006e
      ETH:  9900
      GNO:  9900
      MGN:  0
      Locked MGN:  0
      Will be unlocked MGN:  0
  
  Pool Balances
    EthPool:
      ETH:  0.35764748559167673
      GNO:  0
      MGN:  895.85764748559167673
      Locked MGN:  0
      Will be unlocked MGN:  0
    GnoPool:
      ETH:  0
      GNO:  0.356931474609511211
      MGN:  35.749380528883991814
      Locked MGN:  0
      Will be unlocked MGN:  0
```
MGN stays in the pool when `Coordinator.withdraw...` is called

This PR only demonstrates the behaviour.

### CODE REVIEW

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr7--dxmgnpoolreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr7--dxmgnpoolreact.review.gnosisdev.com

File: trufflescripts/auction.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ async function runStep(step, accounts, contracts, master) {
       await participateInAuction(contracts)
       break
     case 4:
-      await fillAndClearAuctions(contracts, (Math.random() / 10 + 0.02).toFixed(3), master)
+      await fillAndClearAuctions(contracts, 0.02, master)
```
Comment (W3stside) [inline trufflescripts/auction.js:138]: hehehe JS math effed you

**Review by W3stside — APPROVED** [review]
Cool beans

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr7--dxmgnpoolreact.review.gnosisdev.com

**Comment by ghost** [comment]
Travis automatic deployment:
 https://pr7--dxmgnpoolreact.review.gnosisdev.com

**Pull request merged**